python-challenges/email-scheduler/devashmit/scheduler.py

```
@@ -0,0 +1,78 @@
+    if dry_run:
+        print(f"  [DRY RUN] → {to}: {subject[:50]}")
+        logging.info(f"DRY RUN | to={to} subject={subject}")
+        return
+    msg = MIMEMultipart()
+    msg["Subject"] = subject
+    msg["From"] = f"{FROM_NAME} <{SMTP_USER}>"
+    msg["To"] = to
+    msg.attach(MIMEText(body, "plain"))
+    try:
+        with smtplib.SMTP(SMTP_HOST, SMTP_PORT) as s:
+            s.ehlo(); s.starttls(); s.login(SMTP_USER, SMTP_PASS)
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)